### ISSUE TITLE
FileAppendTransaction incorrect content after deserialization

### ISSUE THREAD

**Issue by SvetBorislavov** [hashgraph/hedera-sdk-js#2167]
### Description

Serializing and then deserializing a FileAppendTransaction causes incorrect content to be set.
After the deserialization, the content is sliced to the chunk size.
Ex. If you have a content length of 100_000 after the deserialization, the content will consist only of the first {chunkSize} bytes.

### Steps to reproduce

1. Run this snippet
```
const { FileAppendTransaction } = require('@hashgraph/sdk');

async function run() {
  const contentLength = 632421;
  const content = generateUInt8Array(contentLength);

  const transaction = new FileAppendTransaction()
    .setTransactionValidDuration(180)
    .setContents(content);

  console.log(transaction.contents.length); //632421 (content length)

  const transactionBytes = transaction.toBytes();

  const transactionFromBytes =
    FileAppendTransaction.fromBytes(transactionBytes);

  console.log(transactionFromBytes.contents.length); //4096 (default chunk size)
}
run();

function generateUInt8Array(length) {
  const uint8Array = new Uint8Array(length);

  for (let i = 0; i < length; i++) {
    uint8Array[i] = Math.floor(Math.random() * 256);
  }

  return uint8Array;
}
```

### Hedera network

other

### Version

v2.42.0-beta.4

### Operating system

None

**Comment by SvetBorislavov** [comment]
Currently testing with version `2.52.0-beta.3` and the following issue has been found:
After serializing and deserializing the `FileAppendTransaction` the following properties have been reset to their default values: `transactionId`, `maxChunks`, `chunkSize`.
To reproduce, please run the following snippet:
```
/* imports */
const {
  Client,
  AccountId,
  Transaction,
  FileAppendTransaction,
  TransactionId,
  Timestamp,
  Hbar,
} = require('@hashgraph/sdk');

async function run() {
  const accountId = AccountId.fromString('0.0.2159149');
  const client = Client.forTestnet();

  // Generate random contents
  const contents = generateUInt8Array(10 * 1024);

  // Create a File Append Transaction
  let transaction = new FileAppendTransaction()
    .setTransactionId(
      TransactionId.withValidStart(accountId, Timestamp.fromDate(new Date()))
    )
    .setMaxChunks(999)
    .setChunkSize(1024)
    .setFileId('0.0.111')
    .setMaxTransactionFee(Hbar.fromTinybars(3333333))
    .setTransactionMemo('File Append Test')
    .setTransactionValidDuration(180)
    .setContents(contents);

  // Serialize
  const transactionBytes = transaction.toBytes();

  // Deserialize
  transaction = Transaction.fromBytes(transactionBytes);

  // Verify the bug
  console.log('transactionId', transaction.transactionId.toString());
  console.log('maxChunks', transaction.maxChunks);
  console.log('chunkSize', transaction.chunkSize);
  console.log('fileId', transaction.fileId.toString());
  console.log('maxTransactionFee', transaction.maxTransactionFee.toString());
  console.log('transactionMemo', transaction.transactionMemo);
  console.log('transactionValidDuration', transaction.transactionValidDuration);
  console.log('contents', transaction.contents.length);

  client.close();
}

function generateUInt8Array(length) {
  const uint8Array = new Uint8Array(length);

  for (let i = 0; i < length; i++) {
    uint8Array[i] = Math.floor(Math.random() * 256);
  }

  return uint8Array;
}

run();
```

As you can see from the logs, the properties have incorrect values

**Comment by SimiHunjan** [comment]
Closing as this should be released in `0.52.0`
